internal/resources/controlled_resources_list.yaml

```diff
@@ -25,13 +25,6 @@ resources:
       group: networking.istio.io
       version: v1alpha3
       kind: Gateway
-  - GroupVersionKind:
```

Comment:
Create a follow up card
TODO: Validate that it is not blocking deletion after upgrade from 1.20.3

Reply:
Verified TODO, looks good